sapl/relatorios/views.py

```diff
@@ -1880,6 +1885,75 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class RelatorioVotacoesNominaisView(RelatorioMixin, VotacoesMultiFormatOutputMixin, FilterView):
+    model = VotoParlamentar
+    filterset_class = RelatorioVotacoesNominaisFilterSet
+    template_name = 'relatorios/RelatorioVotacoesNominais_filter.html'
+    relatorio = relatorio_votacao_nominal
+
```

Comment:
Adicionar paginação para melhoria do desempenho:

```suggestion
    paginate_by = 50  # Add pagination for better performance
```